core-bundle/src/Resources/contao/elements/ContentAlias.php

```diff
@@ -38,12 +38,30 @@ public function generate()
 			return '';
 		}
 
+		if (is_a($strClass, ContentProxy::class, true))
+		{
+			if (!empty($this->cssID[1]))
+			{
+				$objElement->classes = array_merge((array) $objElement->classes, [$this->cssID[1]]);
+			}
+
+			/** @var ContentProxy $proxy */
+			$proxy = new $strClass($objElement, $this->strColumn);
+
```

Comment:
```suggestion
$proxy->addFragmentAttributes(['typePrefix' => 'ce_']);

```

Reply:
Didn't find any places where the type prefix is actually used. Maybe we can drop it completely for fragments.

Reply:
I think so too, that's only used for Hybrids